### PR TITLE
JBPM-5457: Start and Stop container buttons change states even though they fail

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/JBPM-5457

Part of an ensemble:
- https://github.com/kiegroup/droolsjbpm-integration/pull/1066
- https://github.com/kiegroup/kie-wb-common/pull/999

### CODE REVIEW

File: kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java [hunk 1]
```diff
@@ -55,189 +54,260 @@ public static KieServerInstanceManager getInstance() {
...
+                                    Container container) {
 
-                        container.setContainerSpecId(containerResource.getContainerId());
-                        container.setContainerName(containerResource.getContainerId());
-                        container.setResolvedReleasedId(containerResource.getResolvedReleaseId() == null ? containerResource.getReleaseId() : containerResource.getResolvedReleaseId());
-                        container.setServerTemplateId(serverTemplate.getId());
-                        container.setStatus(containerResource.getStatus());
-                        container.setMessages(containerResource.getMessages());
+                final ServiceResponse<KieContainerResource> response = client.getContainerInfo(containerSpec.getId());
+                final KieContainerResource containerResource = response.getResult();
```
Comment (mswiderski) [inline kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java]: @karreiro so this is the actual change, is that correct?

Comment (mswiderski) [inline kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java]: essentially we simply return data regardless of the container status, did I got it right?

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java (hunk 1 above)
Comment (karreiro) [inline kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/KieServerInstanceManager.java]: Precisely, @mswiderski :-)

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by karreiro** [comment]
Jenkins retest this

**Comment by karreiro** [comment]
Jenkins retest this

**Comment by manstis** [comment]
@karreiro ```kie-wb-common``` on ```master``` is broken atm.. IDK where the compilation of this PR is failing; but if ```kie-wb-common``` it won't build no matter how many times you try atm.. waiting on @porcelli to fix it :-)

**Comment by karreiro** [comment]
@manstis Thank you. Let's wait for it so :-)

**Comment by karreiro** [comment]
Jenkins retest this

**Comment by karreiro** [comment]
Jenkins retest this

**Comment by karreiro** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
@karreiro could you please rebase as it seems it has conflicts now...

**Comment by karreiro** [comment]
Jenkins retest this

**Pull request merged**